lib/chat/bloc/helper/ChatSummaryProviderHelper.dart

```diff
@@ -1,8 +1,5 @@
-abstract class ChatSummaryProviderHelper<OUT, IN> {
-  OUT getSummary({
-    IN inputModel,
-    String titleValue,
-    String titleText,
-    String actionText,
-  });
+import 'package:flutter/material.dart';
+
+abstract class ChatSummaryProviderHelper {
```

Comment:
providerHelper ? maybe just provider

Reply:
As provider is something that we are adding to Bloc stuff, I'm not really sure what name should we leave here. Could be ChatSummaryProvider if you can confirm it @LeeMatthewHiggins 

thanks!

Reply:
Renamed to ChatMessageViewModelHandler/ChatMessageViewModelHandlerImpl